react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class RegistrationForm extends React.Component {
+constructor(props) {
+  super(props);
+  this.state = {
+    username: '',
+    password: ''
+  };
+  this.handleUsernameChange = this.handleUsernameChange.bind(this);
+  this.handlePasswordChange = this.handlePasswordChange.bind(this);
+  this.handleSubmit = this.handleSubmit.bind(this);
+}
+
+handleUsernameChange() {
+  this.setState({username: event.target.value});
+}
+
+handlePasswordChange() {
+  this.setState({password: event.target.value});
+}
+
+handleSubmit() {
+  event.preventDefault();
+  console.log('State:', this.state);
+}
+
+render() {
+  return (
+    <form>
+      <label>Username
+        <input
+        type={`text`}
```

Comment:
When setting a prop to a string you don't need to use a JavaScript expression `{ }`. For example on the `type` prop you can do:
```JSX
type="text"
```